spec/models/story_spec.rb

```diff
@@ -337,6 +337,53 @@ def fake_response(content, type, code = '200')
     end
   end
 
+  context 'merging stories' do
+    context 'when merging a non-submitter-authored story into a submitter-authored story' do
+      let!(:mgd_user) { create(:user) }
+      let!(:og_user) { create(:user) }
+      let!(:og_story) do
+        create(
+          :story,
+          user: og_user,
+          user_is_author: true
+        ).tap(&:save)
+      end
+      let!(:mgd_story) do
+        create(
+          :story,
+          user: mgd_user,
+          title: 'A Tale of Merging',
+          merged_into_story: og_story,
+          user_is_author: false
+        ).tap(&:save)
```

Comment:
Do you need to call on `.tap(&:save)`?

FactoryBot `create` returns a `saved` instance

```
# Returns a saved User instance
user = create(:user)
```

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#build-strategies